src/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.css';
+import { movieList } from './modules/homepage.js';
+import popup from './modules/popup.js';
+import modalClose from './modules/modalClose.js';
+// import createComment from './modules/app.js';
+
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)